gammapy/scripts/setup_package.py

```diff
@@ -0,0 +1,6 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Add schema validation YAML file for High-level interface API"""
+
+
+def get_package_data():
```

Comment:
Since today ( #2279 ) we maintain package data here:
https://github.com/gammapy/gammapy/blob/c5deffad757a69e26443986a8f0d94beda3d9af8/setup.cfg#L67

Could you please remove setup_package.py again and do the change in this PR?